dget.d

```diff
@@ -15,15 +15,15 @@ void usage()
 
 int main(string[] args)
 {
-    if (args.length == 1) return usage(), 1;
```

Comment:
I had a few in ae:

https://github.com/CyberShadow/ae/commit/d8ef8eb5f01b8c630871c1cfb7a5c3c662ba3c5a
https://github.com/CyberShadow/ae/commit/99585d41635344258a00f72d94461031f4d01afa
https://github.com/CyberShadow/ae/commit/1674d2b91aed3ba53886c92915a76a6e241f5ad8